pinot-controller/src/test/java/org/apache/pinot/controller/helix/core/rebalance/TestZkBasedTableRebalanceObserver.java

```diff
@@ -41,10 +45,13 @@ void testZkObserverTracking() {
     PinotHelixResourceManager pinotHelixResourceManager = mock(PinotHelixResourceManager.class);
     // Mocking this. We will verify using numZkUpdate stat
     when(pinotHelixResourceManager.addControllerJobToZK(any(), any(), any())).thenReturn(true);
+    ControllerMetrics controllerMetrics = Mockito.mock(ControllerMetrics.class);
+    doNothing().when(controllerMetrics).setValueOfTableGauge("dummy", ControllerGauge.TABLE_REBALANCE_IN_PROGRESS, 0);
+    doNothing().when(controllerMetrics).setValueOfTableGauge("dummy", ControllerGauge.TABLE_REBALANCE_IN_PROGRESS, 1);
```

Comment:
Do we actually need this? Isn't do nothing the default mockito behavior?

Reply:
Yes, removed.